src/profile-logic/marker-data.js

```diff
@@ -1313,6 +1315,18 @@ export function removePrefMarkerPreferenceValues(
   return { ...payload, prefValue: '' };
 }
 
+/**
+ * Sanitize FileIO marker's filename property if it's non-empty.
+ */
+export function sanitizeFileIOMarkerFilenamePath(
+  payload: FileIoPayload
+): FileIoPayload {
+  return {
+    ...payload,
+    filename: removeFilePath(payload.filename),
```

Comment:
I noticed in your deploy preview that some FileIO markers don't have a filename, is that true? Or is that we get the empty string sometimes?
(optional) Your code works with empty strings but we may want to make it more explicit by checking this case explicitely.